Content.Server/SS220/ItemOfferVerb/Systems/ItemOfferSystem.cs

```diff
@@ -41,10 +41,25 @@ public override void Update(float frameTime)
                     _alerts.ClearAlert(uid, AlertType.ItemOffer);
                     _entMan.RemoveComponent<ItemReceiverComponent>(uid);
                 }
+                //FunTust: added a new variable responsible for whether the object is still in the hand during transmission
+                bool foundInHand = false;
```

Comment:
```suggestion
                var foundInHand = false;
```
локальные переменные лучше писать через var, если не нужна строгая типизация